apps/site/src/components/pages/wordpress/vote-cast-modal.tsx

```diff
@@ -0,0 +1,251 @@
+import { faDiscord, faTwitter } from "@fortawesome/free-brands-svg-icons";
+import {
+  faCircleCheck,
+  faComment,
+  faTimes,
+  faZap,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  Box,
+  IconButton,
+  modalClasses,
+  Typography,
+  useTheme,
+} from "@mui/material";
+import { FunctionComponent, ReactNode } from "react";
+
+import { EnvelopeIcon, FontAwesomeIcon, GithubIcon } from "../../icons";
+import { LinkButton } from "../../link-button";
+import { Modal } from "../../modal/modal";
+
+export interface CTASectionProps {
+  title: string;
+  subtitle: string;
+  icon: IconDefinition;
+  textColor: string;
+  backgroundColor: string;
+  borderColor: string;
+  children: ReactNode;
+}
+
+export const CTASection: FunctionComponent<CTASectionProps> = ({
+  title,
+  subtitle,
+  icon,
+  textColor,
+  backgroundColor,
+  borderColor,
+  children,
+}) => (
+  <Box
+    sx={{
+      background: backgroundColor,
+      alignItems: "start",
+      padding: "12px 16px",
+      borderRadius: 1,
+      width: 1,
+      justifyContent: "start",
+      marginTop: 1.25,
+      border: `1px solid ${borderColor}`,
+    }}
+  >
+    <Box display="flex" gap={2}>
+      <FontAwesomeIcon
+        icon={icon}
+        sx={{
+          fontSize: 16,
+          color: textColor,
+          mt: 0.75,
+        }}
+      />
+
+      <Box>
+        <Typography
+          variant="bpBodyCopy"
+          sx={{
+            color: textColor,
+            fontWeight: 500,
+            fontSize: 15,
+            textAlign: "start",
+            line: 1.5,
+          }}
+        >
+          {title}
+        </Typography>
+        <Typography
+          variant="bpBodyCopy"
+          sx={{
+            fontWeight: 400,
+            fontSize: 15,
+            textAlign: "start",
+            line: 1.5,
+            mb: 1.5,
+          }}
+        >
+          {subtitle}
+        </Typography>
+
+        <Box display="flex" gap={1.5}>
+          {children}
+        </Box>
+      </Box>
+    </Box>
+  </Box>
+);
+
+export interface ActionButtonProps {
+  label: string;
+  icon: ReactNode;
+  href: string;
+}
+
+export const ActionButton: FunctionComponent<ActionButtonProps> = ({
+  label,
+  icon,
+  href,
+}) => (
+  <LinkButton
```

Comment:
[optional]: It's best to use `{ return <JSX /> }` in all components. This will reduce diffs when adding / removing hooks.